tests/test_webdriver_phantomjs.py

```diff
@@ -69,6 +70,70 @@ def test_can_work_on_popups(self):
         # we can implement this test
         pass
 
+class PhantomJSExecutablePathTest(WebDriverTests, unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        lib_path = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..', 'lib'))
```

Comment:
@bmcculley aren't you forgetting to commit this `lib` folder? I don't see it anywhere.